src/aggregates/vector_selector.rs

```diff
@@ -814,55 +813,21 @@ mod tests {
                 .expect("SQL query failed");
 
         // Assert that we're running in parallel mode.
-        // Note: this query plan is specific to PG14.
+        let top_level_plan = parallel_plan.0[0]["Plan"].clone();
```

Comment:
This is great!

Reply:
Yes! This is so much better than the hacky thing I had implemented!